templates/sway-test-rs/README.md

```diff
@@ -6,6 +6,6 @@ testing within an existing Sway project.
 This template is designed for Rust developers who wish to test integration of
 their Rust application and Sway code.
 
-See the "Testing with Rust" chapter of [the Sway
-Book](https://fuellabs.github.io/sway/v0.25.0/) for a thorough guide on how to
+See the [Testing with Rust](https://fuellabs.github.io/sway/master/testing/testing-with-rust.html) chapter of [the Sway
+Book](https://fuellabs.github.io/sway) for a thorough guide on how to
```

Comment:
I'm suggesting this only because the two links now point to different books which is odd.
```suggestion
Book](https://fuellabs.github.io/sway/master) for a thorough guide on how to
```